lib/wakes/middleware/redirector.rb

```diff
@@ -17,6 +17,8 @@ def calculate_target_and_redirect(request)
         if target = TargetCalculator.new(request).run
           redirect(request.fullpath, target)
         end
+      rescue StandardError
```

Comment:
Why not catch a more specific error here?

Reply:
I can go either way on that. My thinking was: are there errors that we _wouldn't_ want to catch and switch to this behavior on?

Reply:
I think errors that we'd want to know about/surface. If this were part of the app, I'd say just report in sentry and leave it at that. Is there something we can do in wakes?

Reply:
mmm good point. not really. it does not have sentry integration. I will make the error more specific